src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import { getNumbers } from "../../utils";
+import cn from "classnames";
+
+type Props = {
+  currentPage: number;
+  items: string[];
+  perPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  items,
+  perPage,
+  onPageChange,
+}) => {
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
```

Comment:
Consider disabling the default behavior of the anchor tag in 'handlePrevPage' to prevent the URL from changing when it's clicked. You can do this by accepting the event parameter in the function and calling `event.preventDefault()`.